pages/home/components/ReImagineYourDailyCommute.tsx

```diff
@@ -18,31 +18,33 @@ const commuteCards: ICommuteCard[] = [
 		highlight: "shopping",
 		img: shopping.src,
 		bgcolor: "#8DD8D7",
-		textColor: "#67CBC9",
+		description:
+			"illustration of multiple shopping bags in different sizes, shapes and colors.",
```

Comment:
You don't need to write "illustration of" or "image of"
refer https://youtu.be/-jn9aaNn8_I to write great alt texts!

Your alt texts must have the word `WalkingPal` and rephrase you sentence such that Search Engines can know about how this image related to the website.

In this case I think we should write
"Go for a shopping with your WalkingPal buddy"
> if you can think of a better sentence, you should write that